build/config.props

```diff
@@ -5,7 +5,7 @@
     <IsEscrowMode>true</IsEscrowMode>
     <MajorNuGetVersion Condition="'$(MajorNuGetVersion)' == ''">4</MajorNuGetVersion>
     <MinorNuGetVersion Condition="'$(MinorNuGetVersion)' == ''">6</MinorNuGetVersion>
-    <PatchNuGetVersion Condition="'$(PatchNuGetVersion)' == ''">0</PatchNuGetVersion>
+    <PatchNuGetVersion Condition="'$(PatchNuGetVersion)' == ''">1</PatchNuGetVersion>
```

Comment:
Could you also check if the target VS branch needs to change for 4.6.1?

Reply:
@PatoBeltran's gonna figure that out, as he's on hot seat. 
We're not sure. 

The idea is create the branch, merge the 3 fixes that need merging first.

Reply:
my point was that he shouldn't have to generate another build to run with the insertion